### ISSUE TITLE
Train/Test Dataset Split and Preprocessing

### ISSUE THREAD

**Issue by tungnthust** [JuliaWolleb/diffusion-anomaly#16]
I want to reproduce your results experimented on BRATS20 dataset reported in your paper. However, I have some troubles in processing that dataset. Could you share the way you split train/test dataset (may be a list of patient ids for each set) and how you normalize raw data into range between 0 and 1 ?

**Comment by ChyaZhang** [comment]
Me too. I need the split list as well

**Comment by tungnthust** [comment]
@JuliaWolleb Could you kindly share hyperparmeters for training classifier for BRATS dataset (i.e learning_rate, batch_size, anneal_lr, weight_decay, dropout) ? I tried your training settings in README, but it does not produce good results (much worse than your pretrained classifier checkpoint you provided). I think the hyperparameters in TRAINING_FLAGS are used for just training diffusion model, not for the classifier model. Since I refered to classifer hyperparmeters used in openai/guided_diffusion repository, it is completely different from your settings cited in README.

**Comment by JuliaWolleb** [comment]
yes, for training the classifier, you need the CLASSIFIER_FLAGS and TRAIN_FLAGS of the README file. The default values are given in the classifier_defaults https://github.com/JuliaWolleb/diffusion-anomaly/blob/509643c3c15fc5af7d4712aa7b1e8ba8a06214ff/guided_diffusion/script_util.py#L28
 Then it should work fine.

**Comment by waiterxiaoyy** [comment]
Hi, I wrote a data set preprocessing code now, you can see if it can be used, please visit: https://github.com/JuliaWolleb/diffusion-anomaly/issues/19
